cspm/rn/prisma-cloud-compute-release-information/prisma-cloud-compute-known-issues.adoc

```diff
@@ -55,6 +55,11 @@ With this fix, the correct AWS product ARN for US and China regions are used.
 //added Kepler Update 2
 |The userinput:[--tarball] option in twistcli does not scan for compliance checks. Currently, only vulnerabilities are detected successfully.
 
+//CWP-47729
```

Comment:
@rcaballeromx, I would write the latest blurbs on the top like I rearranged in [22.12 Known Issues](https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-12/prisma-cloud-compute-edition-release-notes/release-information/known-issues-22-12).
This will be easier for the user rather than scrolling down all the way to the end for the latest fixes.
WDYT?
Cc: @Pubs-MV

Reply:
correct. We want the latest fixes on the top. But the rest of the known issues are chronological.

Reply:
@Pubs-MV Moved.